feature-libs/organization/administration/core/services/org-unit.service.ts

```diff
@@ -37,6 +37,16 @@ export class OrgUnitService {
     protected userIdService: UserIdService
   ) {}
 
+  clearUsersData(
```

Comment:
```suggestion
  clearAssignedUsersList(
```

Current name suggests that it would remove everything from the store related to users, but we only remove list of assigned users